Migration.md

```diff
@@ -558,6 +558,7 @@ ABI was broken for `sdf::Element`, and restored on version 11.2.1.
 ### Additions
 
 1. **world.sdf**: A joint can be specified directly in a world.
+1. **world.sdf**: Merge-includes are now allowed in worlds. The included models must not contain links or grippers.
```

Comment:
nit Consider qualifying:

"The included models must not contain **top-level** links or grippers."

Reply:
Done in https://github.com/gazebosim/sdformat/pull/1233/commits/f6a975222f4f6688600be7f364c09ec497d97d62

Reply:
Thanks!